iris-mpc/bin/server_hawk.rs

```diff
@@ -1360,63 +1360,64 @@ async fn server_main(config: Config) -> eyre::Result<()> {
                 })
                 .collect::<eyre::Result<Vec<_>>>()?;
 
-            let mut tx = store_bg.tx().await?;
-
-            store_bg
-                .insert_results(&mut tx, &uniqueness_results)
-                .await?;
+            // Initialise dB transaction.
+            let mut tx = iris_pgres_store_bg.tx().await?;
 
-            // TODO: update modifications table to store reauth and deletion results
+            // Persist Iris postgres store state changes.
+            if config_bg.mode_of_deployment == ModeOfDeployment::STANDARD {
```

Comment:
It seems like this parens is never closed. Otherwise, let's resolve the merge conflicts / fmt / etc. and make a final decision on this?